proto/compliance/tx.proto

```diff
@@ -85,4 +86,33 @@ message MsgProvisionModel {
 message MsgProvisionModelResponse {
 }
 
+message MsgUpdateComplianceInfo {
+  string creator = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
+  int32 vid = 2 [(gogoproto.moretags) = "validate:\"gte=1,lte=65535\""]; 
+  int32 pid = 3 [(gogoproto.moretags) = "validate:\"gte=1,lte=65535\""];
+  uint32 softwareVersion = 4;
+  string softwareVersionString = 5 [(gogoproto.moretags) = "validate:\"omitempty,max=64\""];
+  string certificationType = 6 [(gogoproto.moretags) = "validate:\"required,max=64\""];
+  uint32 cDVersionNumber = 7 [(gogoproto.moretags) = "validate:\"omitempty,gte=0,lte=65535\""]; 
+  uint32 softwareVersionCertificationStatus = 8; 
+  string date = 9; 
+  string reason = 10 [(gogoproto.moretags) = "validate:\"omitempty,max=102400\""]; 
+  string owner = 11 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string cDCertificateId = 12 [(gogoproto.moretags) = "validate:\"omitempty,max=64\""];
```

Comment:
`cDCertificateId` is tricky (see https://github.com/zigbee-alliance/distributed-compliance-ledger/issues/356 and https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/x/compliance/keeper/msg_server_certify_model.go#L100).
The corresponding indices need to be updated if it's changed.